docs/quickstart.rst

```diff
@@ -202,13 +202,13 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     # limit prune's operation to this machine's archives and not apply to
     # other machines' archives also:
 
-    borg prune                          \
-        --list                          \
-        --prefix '{hostname}-'          \
-        --show-rc                       \
-        --keep-daily    7               \
-        --keep-weekly   4               \
-        --keep-monthly  6               \
+    borg prune                      \
+    --list                          \
+    --glob-archives '{hostname}-'   \
```

Comment:
needs to be `--glob-archives '{hostname}-*'`.

also maybe rather keep the indentation of the lines with the options, so it is better to see that they belong to the first line with the beginning of the command.

In line 211, remove all spaces before the line end, so there are no trailing spaces.

also, fix the create command in the same way.

Reply:
Okay, I'll fix it. But I didn't get you here: `keep the indentation of the lines with the options, so it is better to see that they belong to the first line with the beginning of the command.`

Reply:
Okay, I'll fix it. Thanks